dev/dll/SharedHelpers.cpp

```diff
@@ -286,7 +286,11 @@ winrt::Rect SharedHelpers::ConvertDipsToPhysical(winrt::UIElement const& xamlRoo
 {
     try
     {
+#if defined(BUILD_WINDOWS)
```

Comment:
I was expecting to see a rewrite in this function to use runtime conditional, is that coming later?

Reply:
weird. i did the change but it looks like it did not get saved.